portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -28,5 +28,10 @@
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     response.setContentType("text/html;");
     response.getWriter().println("<h1>Hello world!</h1>");
+    response.getWriter().println("Hello Rosanna!");
+    //the code below shows the same thing. with p tags and without p tags show the  
+    //same font and size
```

Comment:
Pick one; leave the other out. It's generally considered bad practice to leave in commented out code.